qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -539,6 +547,10 @@ def _require_alignment(property_set):
     return scheduling
 
 
+@deprecate_func(
+    additional_msg="Instead, use ``get_vf2_limits``.",
+    since="0.24.0",
+)
```

Comment:
This is a bit too quick, we can't deprecate it until after we're included the alternative in the release, see: https://qiskit.org/documentation/deprecation_policy.html#removing-a-feature so can you remove the decorator and the release note for this. We can do this for 0.25.0, but not for this PR which we'll need to backport to 0.24.0 (alternatively you can set `pending=True` on the decorator and remove the release not to downgrade the warning emitted to `PendingDeprecationWarning`)

Reply:
Ah that's right, thanks for the reminder. I removed the decorator as well as the release note in 5ef7a94.